_posts/2015-06-11-social-buttons-tutorial.md

```diff
@@ -21,4 +21,14 @@ social:
     url: "/feed.xml"
 </pre>
 
+The titles of the social link are used to find the icon in font awesome css. 
+To look for an icon, search for one at the [Font Awesome icon gallery](https://fortawesome.github.io/Font-Awesome/icons/) and put the icon name (without fa-) in the title.
+
+If you want to add the Twitter icon in the social buttons, the following code snippet will help.
```

Comment:
Actually it's easier, the social buttons are configured from the _config.yml

So we can just provide the url which lists the available icons instead of just saying "There are plenty of social networks, just try them out and see its rendering in the browser " :wink: